enclave/core/tracee.c

```diff
@@ -0,0 +1,205 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#include <openenclave/bits/safecrt.h>
+#include <openenclave/bits/safemath.h>
+#include <openenclave/bits/types.h>
+#include <openenclave/enclave.h>
+#include <openenclave/internal/calls.h>
+#include <openenclave/internal/enclavelibc.h>
+#include <openenclave/internal/report.h>
+#include <openenclave/internal/sgxtypes.h>
+#include <openenclave/internal/trace.h>
+#include <openenclave/internal/utils.h>
+
+static oe_log_filter_t g_active_log_filter = {0};
+static char g_logging_filename[MAX_FILENAME_LEN];
+static bool g_debug_allowed_enclave = false;
+
+const char* get_filename_from_path(const char* path)
```

Comment:
oe functions should have an oe_ prefix or if static, should have _ prefix.

Reply:
renamed all newly added static function to have _ prefix.

Reply:
@anakrish is correct. @soccerGB -- Static functions should be prefixed with  the \_oe\_ prefix and oe functions with oe_ as these are part of the core.

Reply:
based on the recommendation from Mike, _ is the prefix for all static vars and routines.